changelog.d/ea-208.fixed

```diff
@@ -0,0 +1,3 @@
+Fixed a bug with pre-filtering introduced in 1.42.0 that caused significant slowdowns,
+particularly for Kotlin repos. Kotlin repos running default pro rules may see a 30 minute
```

Comment:
Simple curiosity, but do we have any clue why particularly for Kotlin repos?

Reply:
There are two Kotlin rules that really blow up

Reply:
I'm going to look into lowering the threshold to abort